### PR TITLE
Lovelace Menu Edits

### PR DESCRIPTION
**Pull request by zsarnett**
## Proposed change

image
image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -3178,7 +3178,7 @@
           "help": "Help",
           "start_conversation": "Start conversation",
           "reload_resources": "Reload resources",
-          "exit_edit_mode": "Exit UI edit mode",
+          "exit_edit_mode": "Done",
```
Comment (zsarnett) [inline src/translations/en.json:3181]: Should I create a new translation? Other languages are going to be a longer title for this

**Comment by spacegaier** [comment]
I think this PR would fix this issue, right? https://github.com/home-assistant/frontend/issues/9826

**Comment by matthiasdebaat** [comment]
> I think this PR would fix this issue, right? https://github.com/home-assistant/frontend/issues/9826

Yeah! 😊

File: src/panels/lovelace/hui-root.ts [hunk 2]
```diff
@@ -377,30 +375,36 @@ class HUIRoot extends LitElement {
...
-                        )}
-                      >
-                        ${this.hass!.localize("ui.panel.lovelace.menu.help")}
-                        <ha-svg-icon
-                          slot="graphic"
-                          .path=${mdiHelp}
-                        ></ha-svg-icon>
-                      </mwc-list-item>
-                    </a>
+                    ${this._editMode
```
Comment (balloob) [inline src/panels/lovelace/hui-root.ts:383]: This doesn't match your screenshot. This is only showing the help link _inside_ the overflow menu when in edit mode.

Shouldn't we just kill the overflow menu in edit mode?

File: src/translations/en.json (hunk 1 above)
Comment (balloob) [reply src/translations/en.json:3181]: It's fine.

File: src/panels/lovelace/hui-root.ts (hunk 2 above)
Comment (spacegaier) [reply src/panels/lovelace/hui-root.ts:383]: Assuming you mean this: 
image

Why kill it? I think the four entries in the edit mode overflow menu are worthwhile and should be kept.

Comment (balloob) [reply src/panels/lovelace/hui-root.ts:383]: oh my bad. forgot about those.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**